recipes/medical_imaging.ipynb

```diff
@@ -6,25 +6,21 @@
    "source": [
```

Comment:
It would be nice to add the 'open in colab' button on top of the notebook (I think the code below is OK, but you better check)
```
{
            "cell_type": "markdown",
            "metadata": {
                "colab_type": "text",
                "id": "view-in-github"
            },
            "source": [
                "<a href=\"https://colab.research.google.com/github/kili-technology/kili-python-sdk/blob/main/recipes/Update medical_imaging.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
            ]
        }
```

Reply:
Done. Pls verify